changelog.md

```diff
@@ -1,3 +1,20 @@
+## Changes from 1.5.6 to 1.5.7
+Bugfix release
```

Comment:
I think it is not a bugfix release anymore, because preferential GPU scheduling is included.